.coveragerc

```diff
@@ -4,16 +4,17 @@ omit =
     */lib/python*/site-packages/*
     */lib/python*/*
     /usr/*
-    setup.py
+    */setup.py
     # TODO(rhcarvalho): this is used to ignore test files from coverage report.
     # We can make this less generic when we stick with a single test pattern in
     # the repo.
     */conftest.py
     */test_*.py
     */*_tests.py
+    */test/*
 
 [report]
-fail_under = 25
+fail_under = 29
```

Comment:
@ashcrow I wouldn't get too excited, the coverage requirement for the separate utils tests was 73% :)